docs/FHSS_SHAPING.md

```diff
@@ -0,0 +1,43 @@
+# mLRS Documentation: FHSS Shaping #
```

Comment:
Should there be a definition for FHSS?

Reply:
I'm not sure. Not sure my change makes it better.

Reply:
Change on Line 5 works for me.